crates/miden-lib/src/account/faucets/mod.rs

```diff
@@ -220,6 +223,36 @@ impl TryFrom<&Account> for BasicFungibleFaucet {
 // FUNGIBLE FAUCET
 // ================================================================================================
 
+/// Extension trait for fungible faucet accounts. Provides methods to access the fungible faucet
+/// account's reserved storage slot.
+pub trait FungibleFaucetExt {
+    const ISSUANCE_ELEMENT_INDEX: usize;
+    const ISSUANCE_STORAGE_SLOT: u8;
+
+    /// Returns the amount of tokens (in base units) issued from this fungible faucet.
+    ///
+    /// # Errors
+    /// Returns an error if the account is not a fungible faucet account.
+    fn get_faucet_issuance(&self) -> Result<Felt, FungibleFaucetError>;
```

Comment:
nit: I wonder if we should say just `get_issuance()` or maybe `get_token_issuance()`.